api/cl_ext_migrate_memobject.asciidoc

```diff
@@ -0,0 +1,35 @@
+// Copyright 2024 The Khronos Group Inc.
+// SPDX-License-Identifier: CC-BY-4.0
+
+include::{generated}/meta/{refprefix}cl_ext_migrate_memobject.txt[]
+
+=== Other Extension Metadata
+
+*Last Modified Date*::
+    2024-07-17
+*IP Status*::
+    No known IP claims.
+
+=== Description
+
+Precursor to {clEnqueueMigrateMemObjects}.
+
+=== New Commands
+
+  * {clEnqueueMigrateMemObjectEXT}
```

Comment:
We don't currently define anchors for any of these EXT commands, types, or enums, so they're currently dead links in the spec.  I suppose we could add these to the spec and define anchors for them, but given that there are no devices that support this extension in the opencl.gpuinfo.org database, maybe the best thing to do is to just link to the extension doc like most of the other EXT extensions in this PR and call things good?

Reply:
There was no mention of this extension in the extension doc either. We may never have had a spec document for it. The need to include an entry in the extension appendix was driven by its existence in the XML. As soon as we start automatically generating the list of new commands, types, etc in the extension appendix the dead links will reappear. The description I've added does link to the equivalent core functionality which AFAICT and sadly is the best documentation we've ever had for this extension. My inclination is to keep the document as is but maybe create an issue to add the descriptions to the spec, maybe as aliases of the core functionality if we can. Note that we'll have a similar problem for cL_ext_device_fission.

Reply:
There is a spec for this extension, though it's quite old (and still plaintext): https://registry.khronos.org/OpenCL/extensions/ext/cl_ext_migrate_memobject.txt

I think if we start automatically generating the list of new commands / types / etc. in the extension appendices I'd prefer to have a "skip list" for older extensions like these where the ROI just isn't high enough to pull them into the spec.  Regardless, for now, I'd really prefer not to knowingly introduce "broken links" into the spec.  If we want to include these EXT commands, types, and enums then I think we should also add appropriate anchors for them, also.

Reply:
Right, I hadn't looked at the registry. Good to know.

We could add a "skip list" or we could do the integration. Unless the behaviour specified in the extension diverges from what we have in core, it should be easy to just add anchors and aliases and get rid of more cruft from the registry.

For now, I've removed the dangling links. We'll have to deal with the issue when we work on #1165.